[debug] [2020-04-10T04:44:29.238Z] ----------------------------------------------------------------------
[debug] [2020-04-10T04:44:29.240Z] Command:       /usr/local/bin/node /usr/local/bin/firebase init
[debug] [2020-04-10T04:44:29.240Z] CLI Version:   8.0.2
[debug] [2020-04-10T04:44:29.240Z] Platform:      darwin
[debug] [2020-04-10T04:44:29.240Z] Node Version:  v12.13.0
[debug] [2020-04-10T04:44:29.241Z] Time:          Fri Apr 10 2020 00:44:29 GMT-0400 (Eastern Daylight Time)
[debug] [2020-04-10T04:44:29.241Z] ----------------------------------------------------------------------
[debug] [2020-04-10T04:44:29.241Z] 
[debug] [2020-04-10T04:44:29.250Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-10T04:44:29.318Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async GoogleAuth.getClient (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (/usr/local/lib/node_modules/firebase-tools/node_modules/google-auth-library/build/src/auth/googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
